### PR TITLE
Add basic 'pathLength' tests on shapes.

### PR DESCRIPTION
**Pull request by Tavmjong**
Tests address issue https://github.com/w3c/svgwg/issues/382

### CODE REVIEW

File: svg/shapes/reftests/pathlength-002.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:html="http://www.w3.org/1999/xhtml">
+  <g id="testmeta">
+    <title>Test of 'pathLength' on shapes.</title>
+    <html:link rel="author"
+          title="Tavmjong Bah"
+          href="http://tavmjong.free.fr"/>
+    <html:link rel="help"
+          href="https://www.w3.org/TR/SVG2/shapes.html#RectElement"/>
```
Comment (ewilligers) [inline svg/shapes/reftests/pathlength-002.svg]: circle?

File: svg/shapes/reftests/pathlength-003.svg [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:html="http://www.w3.org/1999/xhtml">
+  <g id="testmeta">
+    <title>Test of 'pathLength' on shapes.</title>
+    <html:link rel="author"
+          title="Tavmjong Bah"
+          href="http://tavmjong.free.fr"/>
+    <html:link rel="help"
+          href="https://www.w3.org/TR/SVG2/shapes.html#RectElement"/>
```
Comment (ewilligers) [inline svg/shapes/reftests/pathlength-003.svg]: circle?

Comment (Tavmjong) [reply svg/shapes/reftests/pathlength-003.svg]: Fixed.

**Comment by ewilligers** [comment]
In Firefox 62.0b3 the ref and non-ref images look the same.

In Edge/Safari, dashed paths appear in 1 and 2 but not 1-ref or 2-ref.

In Blink/Edge/WebKit, text appears around the circles in 3-ref but not 3.
(In Edge, text appears at the top left of 3 separately from the circles. In Blink/WebKit, the text does not appear in 3.)

**Review by ewilligers — APPROVED** [review]

**Pull request merged**